src/sql/azdata.proposed.d.ts

```diff
@@ -750,6 +754,7 @@ declare module 'azdata' {
 		getQueryRows(rowData: QueryExecuteSubsetParams): Thenable<QueryExecuteSubsetResult>;
 		disposeQuery(ownerUri: string): Thenable<void>;
 		saveResults(requestParams: SaveResultsRequestParams): Thenable<SaveResultRequestResult>;
+		setQueryExecutionOptions(ownerUri: string, options: QueryExecutionOptions): Thenable<void>;
```

Comment:
>setQueryExecutionOptions [](start = 2, length = 24)

What about getQueryExecutionOptions?

Reply:
I don't have a use case for needing to get the current execution options for an editor, so I haven't plumbed through that API now.  

What use case do you have in mind where an extension will want to get the current execution options for an edition?  I guess it would make the API more symmetrical, but it does add another API to maintain.

Reply:
Yeah this is more of a symmetry thing. And if extensions are setting the options I could see the case where they'd also want to know "what options are currently set for this query window".

But given that there's no specific need for that right now I guess it's fine to put off until we have an actual demand for it.